module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,56 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressed;
+    for (const auto& line : bitmap) {
+        uint8_t count{};
+        uint8_t pixel = line.at(0);
```

Comment:
Just use fornt(). This is array, so there are always values form 0 do size -1.

Reply:
front() used